packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -1977,6 +1990,7 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
                         declaration={declaration}
                         submitDeclarationAction={submitClickEvent}
                         rejectDeclarationAction={rejectDeclarationClickEvent}
+                        hasErrorsOnFields={hasValidationErrors}
```

Comment:
I'm testing this and have an invalid date for child's DoB but this value is `false`. If I fix the error it's `true` 🤔

Reply:
Yeah, fixed it on the latter commit